inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -1335,7 +1337,7 @@ GnaWaitStatus GNAPlugin::WaitFor(uint32_t request_idx, int64_t millisTimeout) {
             details::product(++std::begin(exportOutputDims), std::end(exportOutputDims));
 
         auto transpose_output_info = transpose_outputs_info.find(outputBlobIt.first);
-        if (transpose_output_info != std::end(transpose_outputs_info)) {
+        if (transpose_output_info != std::end(transpose_outputs_info) && transpose_output_info->second[0].transpose) {
```

Comment:
transpose flag may be true not in the first vector element, in this case we still need to transpose some parts of the output